conflictlog/pool.go

```diff
@@ -236,6 +236,10 @@ func (pool *connPool) Get(bucketName string, timeout time.Duration) (conn io.Clo
 	}
 
 	conn, err = pool.newConnFn(bucketName)
+	if err != nil {
+		pool.limiter.Release()
```

Comment:
Shouldn't there be also a `Release()` in before the return in line 235?
If so, should there just be a `defer` call, then?

Or am I mis-reading the code and its intention?